templates/expo-template-bare-minimum/ios/Podfile

```
@@ -22,6 +22,16 @@ target 'HelloWorld' do
   use_frameworks! :linkage => podfile_properties['ios.useFrameworks'].to_sym if podfile_properties['ios.useFrameworks']
   use_frameworks! :linkage => ENV['USE_FRAMEWORKS'].to_sym if ENV['USE_FRAMEWORKS']
 
+  # Flipper support in expo via build properties plugin
+  expo_flipper = FlipperConfiguration.disabled
+  if ENV['FLIPPER_DISABLE'] != "1" && ENV['NO_FLIPPER'] != "1" && defined?(podfile_properties['ios.flipper']) then
```

```suggestion
  if ENV['NO_FLIPPER'] != "1" && defined?(podfile_properties['ios.flipper']) then
```

any reasons to add `FLIPPER_DISABLE`? i didn't see the variable be used anywhere.

---

It was the legacy for those coming from expo-community-flipper. However, I think taking the react-native standardized ENV makes more sense for SDK 48